classes/widgets.php

```diff
@@ -251,27 +251,27 @@ public function form( $instance ) {
 		<p><input type="checkbox" class="checkbox" id="<?php echo esc_attr(  $this->get_field_id( 'show_address' ) ); ?>"
 		          name="<?php echo esc_attr( $this->get_field_name( 'show_address' ) ); ?>"<?php checked( $show_address ); ?> />
 			<label
-				for="<?php echo esc_attr( $this->get_field_id( 'show_address' ) ); ?>"><?php _e( 'Show Address', 'simplemap' ); ?></label><br/>
+				for="<?php echo esc_attr( $this->get_field_id( 'show_address' ) ); ?>"><?php _e( 'Show Address', 'SimpleMap' ); ?></label><br/>
 
 			<input type="checkbox" class="checkbox" id="<?php echo esc_attr( $this->get_field_id( 'show_city' ) ); ?>"
 			       name="<?php echo esc_attr( $this->get_field_name( 'show_city' ) ); ?>"<?php checked( $show_city ); ?> />
 			<label
-				for="<?php echo esc_attr( $this->get_field_id( 'show_city' ) ); ?>"><?php _e( 'Show City', 'simplemap' ); ?></label><br/>
+				for="<?php echo esc_attr( $this->get_field_id( 'show_city' ) ); ?>"><?php _e( 'Show City', 'SimpleMap' ); ?></label><br/>
 
 			<input type="checkbox" class="checkbox" id="<?php echo esc_attr( $this->get_field_id( 'show_state' ) ); ?>"
 			       name="<?php echo esc_attr( $this->get_field_name( 'show_state' ) ); ?>"<?php checked( $show_state ); ?> />
 			<label
-				for="<?php echo $this->get_field_id( 'show_state' ); ?>"><?php _e( 'Show State', 'simplemap' ); ?></label><br/>
+				for="<?php echo $this->get_field_id( 'show_state' ); ?>"><?php _e( 'Show State', 'SimpleMap' ); ?></label><br/>
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 127 characters

<!-- Comment by [SideCI](https://sideci.com) -->